### PR TITLE
Fixed #33805 -- Made admin's many-to-many widgets do not display help text for selecting values when allow_multiple_selected is False.

### PR DESCRIPTION
**Pull request by ankurr0y**
The message help text will now render accordingly.

### CODE REVIEW

File: django/contrib/admin/options.py [hunk 1]
```diff
@@ -324,6 +328,12 @@ def formfield_for_manytomany(self, db_field, request, **kwargs):
             form_field.help_text = (
                 format_lazy("{} {}", help_text, msg) if help_text else msg
             )
+        else:
+            msg = _("Select one element from the field.")
+            help_text = form_field.help_text
+            form_field.help_text = (
+                format_lazy("{} {}", help_text, msg) if help_text else msg
+            )
```
Comment (felixxm) [inline django/contrib/admin/options.py:336]: Help message is not necessary in this case. Please remove this.

**Review by felixxm — COMMENTED** [review]
@ankurr0y Thanks for this patch :+1: Please add a regression test.

File: django/contrib/admin/options.py (hunk 1 above)
Comment (ankurr0y) [reply django/contrib/admin/options.py:336]: Gotcha, I will get on that.

Comment (ankurr0y) [reply django/contrib/admin/options.py:336]: Hey, I'm having some trouble with this unit test.

How do I explicitly pass widget.allow_multiple_selected into the options?

For the test you have to call it from the formfield_for_dbfield function which passes to the aforementioned options.py. However, it is not letting me pass allow_multiple_selected as a parameter.

Comment (ankurr0y) [reply django/contrib/admin/options.py:336]: Nevermind, found it.

**Comment by felixxm** [comment]
@ankurr0y Thanks for updates :+1:

**Pull request merged**